app/pages/HomePage/components/Partners/Partners.tsx

```diff
@@ -7,59 +7,64 @@ import JDCLogo from "./assets/JDCLogo.png";
 import MOHCDLogo from "./assets/MOHCDLogo.png";
 import BenetechLogo from "./assets/BenetechLogo.png";
 import Our415Logo from "../../../../assets/img/Our415_logo-hori.svg";
-
 import BridgeLogo from "./assets/BridgeLogo.png";
+import AcuteCareLogo from "../../../../assets/img/ucsf-acute-care-innovation-center-logo.png";
 
 export const Partners = () => (
   <div className="partners">
-    <ul>
-      <li>
-        <header>Powered By:</header>
-        <a
-          href="http://sheltertech.org"
-          rel="noopener"
-          className="image-container"
-        >
-          <img src={STLogo} alt="ShelterTech" />
-        </a>
-      </li>
-      <li>
-        <header>In Partnership With:</header>
-        <a
-          href="https://sfmohcd.org/"
-          rel="noopener"
-          className="image-container"
-        >
-          <img src={MOHCDLogo} alt="MOHCD" />
-        </a>
-      </li>
-      <li>
-        <header />
```

Comment:
Looking at the CSS and the rendered page carefully, I think I understand what these empty headers were doing before and why you removed them, but I think it's causing things to be misaligned. Interestingly enough, I have my browser set to a narrower width than you (looks like my width is 1363px), and the Bridge logo was already on its own row for me, prior to the changes in this PR.

With the new logo and other CSS changes, I see this subtle, though noticeable pattern where the logos on the first row seem to creep on upwards, almost in a straight, diagonal line:

<img width="1147" alt="Screen Shot 2023-10-22 at 4 43 10 PM" src="https://github.com/ShelterTechSF/askdarcel-web/assets/1002748/bb727287-fd9c-4e56-8e5f-11bf52997657">

I feel that it's a little bit more distracting than the version you see, where the new logo is on the first row, which helps bring the "center of gravity" back down.

Ideally, we really should be putting the header stuff into their own row so that they don't affect the flex layout of the logos, but as a shorter workaround, what do you think about adding a top-padding to all items except for the first one? I think you can do that by adding the following CSS:

```css
.partners li:not(:first-child) {
  padding-top: 30px;
}
```